Formula/ditaa.rb

```diff
@@ -1,13 +1,25 @@
 class Ditaa < Formula
   desc "Convert ASCII diagrams into proper bitmap graphics"
   homepage "http://ditaa.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/ditaa/ditaa/0.9/ditaa0_9.zip"
-  sha256 "d689e933b80b065cd7c349e489cfb8feea69dd3e91ca78931edc6fa6e098e689"
+  url "https://github.com/stathissideris/ditaa/archive/v0.10.tar.gz"
+  sha256 "82e49065d408cba8b323eea0b7f49899578336d566096c6eb6e2d0a28745d63b"
 
-  bottle :unneeded
+  depends_on :ant => :build
+  depends_on :java
 
   def install
-    libexec.install "ditaa0_9.jar"
-    bin.write_jar_script libexec/"ditaa0_9.jar", "ditaa"
+    # 0.10 Release didn't still calls itself 0.9
+    # Reported upstream at https://github.com/stathissideris/ditaa/issues/14
+    inreplace "build/release.xml", "0_9", "0_10"
+    inreplace "src/org/stathissideris/ascii2image/core/CommandLineConverter.java", "ditaa version 0.9", "ditaa version 0.10"
+
+    mkdir "bin"
```

Comment:
That would actually create `"#{prefix}/bin"` which is a different directory. If you want to use `mkpath` here, it needs to be `(buildpath/"bin").mkpath`.